api/v1/models/profile.py

```diff
@@ -16,9 +16,6 @@ class Profile(BaseTableModel):
     __tablename__ = 'profiles'
 
     user_id = Column(String, ForeignKey('users.id', ondelete="CASCADE"), unique=True, nullable=False)
-    username = Column(String, nullable=True)
-    first_name = Column(String, nullable=True)
-    last_name = Column(String, nullable=True)
```

Comment:
why editing the model

Reply:
also not me ;)

Reply:
@Goketech most of the changes wasnt his.